android/src/main/kotlin/br/com/thyagoluciano/flutterradio/player/RadioService.kt

```diff
@@ -125,11 +125,13 @@ class RadioService(val context: Context) : Player.EventListener, AudioManager.On
             else -> PlaybackStatus.IDLE
         }
 
-        EventBus.getDefault().post(status)
+        if (EventBus.getDefault().hasSubscriberForEvent(String::class.java))
+            EventBus.getDefault().post(status)
```

Comment:
without subscribers it will spam in console:
```
No subscribers registered for event class java.lang.String
No subscribers registered for event class org.greenrobot.eventbus.NoSubscriberEvent
```